pom.xml

```diff
@@ -465,7 +473,7 @@ limitations under the License.
                 <configuration>
                     <java>
                         <googleJavaFormat>
-                            <version>1.7</version>
+                            <version>1.10.0</version>
```

Comment:
nit: It's ideal to follow Flink code format style (AOSP 1.24.0). 
As that would result in changes in many files, we could do it later.

Reply:
Traced in https://issues.apache.org/jira/browse/FLINK-38838.